var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -178,7 +178,7 @@ class Openmpi(AutotoolsPackage):
         'fabrics',
         default=None if _verbs_dir() is None else 'verbs',
         description='List of fabrics that are enabled',
-        values=('psm', 'psm2', 'pmi', 'verbs', 'mxm'),
+        values=('psm', 'psm2', 'verbs', 'mxm'),
```

Comment:
I checked the docs, and I think I was wrong putting `pmi` under fabrics. Thanks for fixing this.